front/lib/api/gens.ts

```diff
@@ -33,7 +33,7 @@ export async function getGensTemplates(
   return templates.map((t) => {
     return {
       name: t.name,
-      instructions: t.instructions,
+      instructions2: t.instructions2,
```

Comment:
why instructions2: t.instructions2 instead of instructions: t.instructions2 here?
You can directly use instructions as key here will prevent you future diff?